File: mp/Web/Models/MessageModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Web.Models
{
	public class MessageModel
	{
		public Int32 MessageId { get; set; }
		public Int32 AuthorId { get; set; }
		public Guid ThreadId { get; set; }
		public String CreatedDate { get; set; }
		public String Text { get; set; }
		public MessageStatus MessageStatus { get; set; }
        public String UserName { get; set; }
    }
}

File: mp/Web/Scripts/spa/models/message_model.js
WebMP.Message = Ember.Object.extend({
	id: undefined,
	authorId: undefined,
	threadId: undefined,
	createdDate: undefined,
	messageStatus: undefined,
	userName: undefined,	
	text: undefined
});

WebMP.Message.reopenClass(WebMP.Api, {
	make: function (obj) {
		return WebMP.Message.create({
			id: obj.messageId,
			authorId: obj.authorId,
			threadId: obj.threadId,
			createdDate: obj.createdDate,
			messageStatus: obj.messageStatus,
			userName: obj.userName,
			text: obj.text,
		});
	}
});
